colossalai/checkpoint_io/hybrid_parallel_checkpoint_io.py

```diff
@@ -264,15 +294,19 @@ def _load(name: str):
                                        strict=strict,
                                        load_sub_module=True)
             del state_dict
+            gc.collect()
```

Comment:
state dict will be collected when this function is done. I think `del` and `gc.collect()` are not essential.

Reply:
modified